rpm/template.spec.em

```diff
@@ -15,6 +15,11 @@ License:        @(License)
 Source0:        %{name}-%{version}.tar.gz
 @[if NoArch]@\nBuildArch:      noarch@\n@[end if]@
 
+%if 0%{?with_weak_deps}
+Recommends:     python%{python3_pkgversion}
```

Comment:
Although it will probably finish in the same install action, why do we prefer `Recommends` instead of making python3 mandatory in all cases, seems strictly needed.

Reply:
It is possible to use this package without Python, though I have no experience doing so:
> Indeed, the use case is to install a subset of packages on a system which doesn't have Python. The user can easily setup the necessary environment variables manually (without Python).

So I thought a weak dependency was appropriate. The default behavior is to install `Recommends` packages unless the user specifically states that they shouldn't be installed.

Reply:
makes sense, thanks.